.travis.yml

```diff
@@ -19,13 +19,17 @@ after_success:
 - docker exec notes /edx/app/notes/notes/.travis/run_coverage.sh
 - codecov
 env:
-  matrix:
+- TOXENV=django111
+- TOXENV=django20
+- TOXENV=django21
+- TOXENV=django22
+matrix:
   - DOCKER_USERNAME=edxbuilder
-  global:
+global:
```

Comment:
These `matrix` and `global` keys need to stay inside `env`.  The environment variables in `global` are set for all workers, the ones in `matrix` define different workers (see https://docs.travis-ci.com/user/environment-variables/#global-variables for details).  So I think what you want here is to move `DOCKER_USERNAME` into `global` and put the different `TOXENV` values under `matrix`.